### PR TITLE
Feature - FixLoadMoreVideoChannel

### PR DESCRIPTION
**Pull request by at-hungpham**

### CODE REVIEW

**Comment by at-bienle** [comment]
:+1: There are no linter violations.

File: FinalProject.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -454,6 +454,7 @@
 		730C85C32341AA0F00B9EE8B /* Custom */ = {
 			isa = PBXGroup;
 			children = (
+				73EFDACE234C1DBA00F53743 /* New Group */,
```
Comment (at-hainguyen3) [inline FinalProject.xcodeproj/project.pbxproj]: Rename folder

File: FinalProject/View/Controller/ChannelLoadMore/ChannelLoadMoreViewController.swift [hunk 2]
```diff
@@ -60,7 +60,7 @@ extension ChannelLoadMoreViewController: UICollectionViewDelegate, UICollectionV
     }
 
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        return CGSize(width: 200, height: 180)
+        return CGSize(width: 405, height: 49)
```
Comment (at-hainguyen3) [inline FinalProject/View/Controller/ChannelLoadMore/ChannelLoadMoreViewController.swift]: Why 405 and 49?

Comment (at-hungpham) [reply FinalProject/View/Controller/ChannelLoadMore/ChannelLoadMoreViewController.swift]: Screen Shot 2019-10-08 at 9 30 51 AM
Screen Shot 2019-10-08 at 9 31 00 AM
để nhìn cho hắn đồng bộ thôi a

File: FinalProject.xcodeproj/project.pbxproj (hunk 1 above)
Comment (at-hainguyen3) [reply FinalProject.xcodeproj/project.pbxproj]: @at-hungpham

File: FinalProject/View/Controller/ChannelLoadMore/ChannelLoadMoreViewController.swift (hunk 2 above)
Comment (at-hainguyen3) [reply FinalProject/View/Controller/ChannelLoadMore/ChannelLoadMoreViewController.swift]: Build lên con iphone 6s xem thử

Comment (at-hungpham) [reply FinalProject/View/Controller/ChannelLoadMore/ChannelLoadMoreViewController.swift]: Screen Shot 2019-10-08 at 9 36 59 AM
Screen Shot 2019-10-08 at 9 37 11 AM

**Pull request merged**